zerver/lib/markdown/api_return_values_table_generator.py

```diff
@@ -137,6 +137,8 @@ def render_table(self, return_values: Dict[str, Any], spacing: int) -> List[str]
             # Test to make sure deprecated keys are marked appropriately.
             if likely_deprecated_parameter(description):
                 assert return_values[return_value]["deprecated"]
+            if "deprecated" in return_values[return_value]:
+                assert likely_deprecated_parameter(description)
```

Comment:
Can you extract this 4-line block to a shared function that we can call from both extensions, e.g. `check_deprecated_consistency(return_values[return_value], description)` ?